tst/TurnConnectionFunctionalityTest.cpp

```diff
@@ -263,6 +263,92 @@ namespace com { namespace amazonaws { namespace kinesis { namespace video { name
         freeTestTurnConnection(&pTurnConnection);
     }
 
+    struct TurnCustomData {
+        PTurnConnection pTurnConnection;
+        PBYTE pBuf;
+        UINT32 bufLen;
+        PKvsIpAddress pKvsIpAddress;
+    };
+
+    PVOID testTurnSendDataRoutine(PVOID arg) {
```

Comment:
Would you mind making this a lambda/keeping it internal to the test if possible?

Just keeps the code a little cleaner for the reader/don't have to worry about others re-using incorrectly.

Reply:
It is only defined in this cpp and not in any .h file, I suppose that would not be reused and just internal to this test file.